### ISSUE TITLE
Bug in which_bays()

### ISSUE THREAD

**Issue by jlashner** [slaclab/pysmurf#417]
## Describe the bug
`which_bays` function fails at UCSD with the error message:
```
cannot connect to smurf_server_s2:AMCc:SmurfApplication:StartupArguments
Traceback (most recent call last):
  File "/sodetlib/hammers/scripts/start_pysmurf_ipython.py", line 37, in <module>
    make_logfile=args.make_logfile
  File "/usr/local/src/pysmurf/python/pysmurf/client/base/smurf_control.py", line 134, in __init__
    **kwargs)
  File "/usr/local/src/pysmurf/python/pysmurf/client/base/smurf_control.py", line 232, in initialize
    self.bays = self.which_bays()
  File "/usr/local/src/pysmurf/python/pysmurf/client/util/smurf_util.py", line 2150, in which_bays
    if f'--disable-bay{bay}' not in smurf_startup_args:
TypeError: argument of type 'NoneType' is not iterable
```
Problem seems to be that `get_smurf_startup_args` is returning None, which this function doesn't like. I patched it temporarily at UCSD by returning `[0,1]` if there are no startup args, but this is probably not what we want for every case. 
## To Reproduce
Run `which_bays()` at UCSD

**Comment by swh76** [comment]
This one confused me because these registers are defined in pysmurf, but I believe the issue is that the registers defined in pysmurf get sourced out of the pysmurf that's packaged with the rogue server you're using, which is not the pysmurf version you're working with (since you're running on the UCSD system in development mode).  We'll need to upgrade you from v4.0.0-rc21 to v4.0.0, since the PR that adds this register (https://github.com/slaclab/pysmurf/pull/314) is in v4.0.0.

**Comment by jlashner** [comment]
Solved by upgrading to v4.0.0
